apps/studio/components/interfaces/Storage/AnalyticBucketDetails/index.tsx

```diff
@@ -140,131 +148,227 @@ export const AnalyticBucketDetails = ({ bucket }: { bucket: Bucket }) => {
+                  label: 'Analytics',
+                  href: `/project/${project?.ref}/storage/analytics`,
+                },
+              ]
+            : []
+        }
+        secondaryActions={config?.docsUrl ? [<DocsButton key="docs" href={config.docsUrl} />] : []}
+      >
+        <ScaffoldContainer bottomPadding>
+          {state === 'loading' && (
+            <ScaffoldSection isFullWidth>
+              <GenericSkeletonLoader />
+            </ScaffoldSection>
+          )}
+          {state === 'not-installed' && (
+            <ExtensionNotInstalled
+              bucketName={bucket.name}
+              projectRef={project?.ref!}
+              wrapperMeta={wrapperMeta}
+              wrappersExtension={wrappersExtension!}
+            />
+          )}
+          {state === 'needs-upgrade' && (
+            <ScaffoldSection isFullWidth>
+              <ExtensionNeedsUpgrade
+                bucketName={bucket.name}
+                projectRef={project?.ref!}
+                wrapperMeta={wrapperMeta}
+                wrappersExtension={wrappersExtension!}
+              />
+            </ScaffoldSection>
+          )}
+
+          {state === 'added' && wrapperInstance && (
+            <>
+              {isStorageV2 ? (
+                <ScaffoldSection isFullWidth>
+                  <ScaffoldHeader className="flex flex-row justify-between items-end gap-x-8">
+                    <div>
+                      <ScaffoldSectionTitle>Tables</ScaffoldSectionTitle>
+                      <ScaffoldSectionDescription>
+                        Analytics tables connected to this bucket.
+                      </ScaffoldSectionDescription>
+                    </div>
+                    <Button type="primary" size="tiny" icon={<Plus size={14} />} onClick={() => {}}>
+                      New table
+                    </Button>
+                  </ScaffoldHeader>
+
+                  <Card>
+                    <Table>
+                      <TableHeader>
+                        <TableRow>
+                          <TableHead className="text-foreground-muted">Name</TableHead>
+                          <TableHead className="text-foreground-muted">Schema</TableHead>
+                          <TableHead className="text-foreground-muted">Created at</TableHead>
+                        </TableRow>
+                      </TableHeader>
+                      <TableBody>
+                        <TableRow>
+                          <TableCell colSpan={4}>
+                            <p className="text-sm text-foreground">No tables yet</p>
+                            <p className="text-sm text-foreground-lighter">
+                              Create an analytics table to get started
+                            </p>
+                          </TableCell>
+                        </TableRow>
+                      </TableBody>
+                    </Table>
+                  </Card>
+                </ScaffoldSection>
```

Comment:
`isStorageV2` is quite different to the current approach. We are showing _tables_ that are connected to namespaces. Not the other way around.

We need to support both, and this is hard to test. Hence why there is this duplicative table and why it’s hardcoded as empty.

I’d love some support getting the real table data in :)